connector/src/main/java/org/geysermc/connector/network/ConnectorServerEventHandler.java

```diff
@@ -106,13 +106,16 @@ public BedrockPong onQuery(InetSocketAddress inetSocketAddress) {
 
     @Override
     public void onSessionCreation(BedrockServerSession bedrockServerSession) {
+        // Initial session created is always the main client, so client id 0
+        final int clientId = 0;
```

Comment:
Just as a general consistency thing across this codebase, unless something absolutely needs to be final (e.g. it won't compile if it's not), it should not be specified as so.

Reply:
Done (in all instances I introduced).